Adyen/UI/List/Icon.swift

```diff
@@ -0,0 +1,65 @@
+//
+// Copyright (c) 2024 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import UIKit
+
+public struct Icon: Hashable {
```

Comment:
Correct me if I'm wrong, do we really need to use the `Icon` from ListItem? This has `location` and `canBeModified` which shouldn't be needed.
Looks like an `imageUrl` property should be enough for this?

Reply:
I have added `imageUrl`  instead of using the Icon and reverted the changes of Icon.